news/2024/2024-10-09-resurrection-cup-2024-concludes.md

```diff
@@ -0,0 +1,316 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/12840110" style="background-image: url('https://a.ppy.sh/users/12840110')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-bsc" href="https://osu.ppy.sh/users/12840110">Phreel</a></p>
+Was it difficult managing all those custom maps and songs?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6248691" style="background-image: url('https://a.ppy.sh/users/6248691')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-bsc" href="https://osu.ppy.sh/users/6248691">Alvearia</a></p>
+Managing around 90 original custom maps sounds like a really tough task to accomplish – but thanks to main host’s/Phreel’s organization plans, selecting small group of knowledgeable people to be a part of Quality Assurance Team and dividing all mappers by divisions was a really smart move to do, and mostly by that, we finished this job successfully. But, nonetheless, there’s plenty of room for improvement, overall quality of pools, which our team, of course, will try absolute best to achieve in the next iteration.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4429612" style="background-image: url('https://a.ppy.sh/users/4429612')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-bsc" href="https://osu.ppy.sh/users/4429612">Luminous Sky</a></p>
+Contrary to my initial expectations, I didn't find managing the custom maps and songs all that difficult, and even sometimes intruded into threads of maps assigned to other members of the mappool quality assurance team because I felt I had to get a decent idea of what the maps are like and not just trust them to fit the norms of a slot or descriptions others give me, or we would get another aim FM2 incident like in Quarterfinals. There was also a period of time where I was really sick for like 2-3 weeks and others rechecked a couple maps after I initially requested some changes to them but felt unable to offer great feedback due to my health. I'm very thankful for that. So, yeah, overall, it wasn't that difficult, but while I of course love how many artists and mappers we gathered for this tourney, I would've still preferred a little fewer customs overall for higher player enjoyment as well.
```

Comment:
```suggestion
Contrary to my initial expectations, I didn't find managing the custom maps and songs all that difficult, and even sometimes intruded into threads of maps assigned to other members of the mappool Quality Assurance Team because I felt I had to get a decent idea of what the maps are like and not just trust them to fit the norms of a slot or descriptions others give me, or we would get another aim FM2 incident like in Quarterfinals. There was also a period of time where I was really sick for like 2-3 weeks and others rechecked a couple maps after I initially requested some changes to them but felt unable to offer great feedback due to my health. I'm very thankful for that. So, yeah, overall, it wasn't that difficult, but while I of course love how many artists and mappers we gathered for this tourney, I would've still preferred a little fewer customs overall for higher player enjoyment as well.
```